lib/Net/Async/Blockchain/ETH.pm

```diff
@@ -234,7 +235,8 @@ sub subscribe {
                     # Skip old checked blocks (e.g. when restarting the node)
                     my $next_block        = await $self->new_blocks_queue->shift;
                     my $next_block_number = Math::BigInt->from_hex($next_block->{params}->{result}->{number});
-                    next if $self->base_block_number and $next_block_number->blt($self->base_block_number);
+                    # to avoid chain re-organization check/emit last 6th block instead of highest block
```

Comment:
Minor suggestion, could we add more details to the comment :pray:  . It's upto you :)